src/lib/layout/shell.svelte

```diff
@@ -19,11 +18,14 @@
     import { page } from '$app/stores';
     import type { Models } from '@appwrite.io/console';
 
-    export let showSideNavigation = false;
     export let showHeader = true;
     export let showFooter = true;
-    export let loadedProjects: Array<NavbarProject> = [];
+    export let showSideNavigation = false;
     export let selectedProject: Models.Project | null = null;
+    export let projectsPromise: Promise<Models.ProjectList> = Promise.resolve({
+        total: 0,
+        projects: []
+    });
```

Comment:
we probably should be a bit more explicit, it should be a promise or undefined. otherwise we dont really know if it was an actual promise or the mock here